.github/workflows/deno.yml

```diff
@@ -0,0 +1,46 @@
+name: Deno
+
+on:
+  push:
+    paths:
+      - deno/**
+    branches:
+      - master
+  pull_request:
+    branches:
```

Comment:
paths should be here too under pull_request.

Reply:
done